pages/checkIn.tsx

```diff
@@ -1,5 +1,44 @@
+import CheckInOutForm from 'components/CheckInOutForm'
+import {
+  Box,
+  Button,
+  Card,
+  CardContent,
+  CardActions,
+  Typography,
+  useMediaQuery,
+} from '@mui/material'
+
 export default function CheckInPage() {
+  const isMobileView = useMediaQuery('(max-width: 600px)')
   return (
-    <h1>Check In</h1>
+    <Box
```

Comment:
remove this box and instead use the MUI [Grid v2](https://mui.com/material-ui/react-grid2/). I've played around with it and will post the code I used to get it looking right.